src/app/file-browser/components/edit-tags/edit-tags.component.html

```diff
@@ -2,8 +2,11 @@
   <pr-tags [tags]="itemTags" (click)="startEditing()" [readOnly]="false" [class.clickable]="canEdit && !isEditing"
     [canEdit]="canEdit" [animate]="isEditing" [isEditing]="isEditing"></pr-tags>
   <div class="input-group" *ngIf="isEditing" @ngIfScaleAnimation [class.has-tags]="allTags?.length">
-    <input type="text" class="new-tag form-control" placeholder="Add new tag"
-      [(ngModel)]="newTagName" (ngModelChange)="onTagType(newTagName)" (keydown.enter)="onInputEnter(newTagName)">
+    <input type="text" class="new-tag form-control" [placeholder]="placeholderText" [(ngModel)]="newTagName"
+    (ngModelChange)="onTagType(newTagName)" (keydown.enter)="onInputEnter(newTagName)">
+    <div *ngIf="newTagInputError" class="input-vertical-error">
+      {{this.tagType === 'keyword' ? "Tag cannot contain ':'" : "Format must be key:value"}}
+    </div>
```

Comment:
This error wasn't in the design, so I just followed the convention used for validation errors in the signup form